src/data/int/basic.lean

```diff
@@ -557,6 +557,33 @@ begin
   tauto,
 end
 
+lemma eq_one_or_neg_one_iff_mul_eq_one {z w : ℤ} :
+  z * w = 1 ↔ z = 1 ∧ w = 1 ∨ z = -1 ∧ w = -1 :=
```

Comment:
This lemma is backwards, `A_iff_B` should name `A ↔ B`

Reply:
OK. Taking inspiration from `mul_eq_zero` (which is a `simp` lemma), I think the *statement* is in the correct form, so I'll change the name (and similarly with the `neg_one` variant).